backend/src/cache/server/mutation.go

```diff
@@ -257,6 +261,7 @@ func prepareMainContainer(pod *corev1.Pod, result string, logger *zap.SugaredLog
 	}
 
 	args := []string{}
+	args = append(args, "printf 'This step output is taken from cache.'")
```

Comment:
Since we are using printf, break two new lines so this message is separate from the cached string.

```suggestion
	args = append(args, "printf 'This step output is taken from cache.\n\n'")
```